martinMutuma/def-team-project
# #164799334 List Team Members

#### What does this PR do?
Add members list endpoint

#### Description of the Tasks to be accomplished
User should be able to view members list

#### How should this be manually tested?
Clone this repo
`$ git clone https://github.com/martinMutuma/def-team-project.git`
CD into the project directory
`$ cd def-team-project`
Check out the branch for this PR
`$ git checkout ft/164799334-view-team-members `
Activate the virtual environment
`$ source venv/bin/activate `

Run the app
`$python manage.py runserver `

on your browser go to http://127.0.0.1:8000

#### What are the relevant pivotal tracker stories?
[#164799334 ](https://www.pivotaltracker.com/story/show/164799334 )

#### Screenshots
N/A
